pkgs/os-specific/linux/kernel-headers/default.nix

```diff
@@ -36,9 +45,12 @@ let
     # Skip clean on darwin, case-sensitivity issues.
     buildPhase = lib.optionalString (!stdenvNoCC.buildPlatform.isDarwin) ''
       make mrproper $makeFlags
-    '' + ''
+    '' + (if stdenvNoCC.hostPlatform.isAndroid then ''
+      make defconfig
+      make headers_install
```

Comment:
Do we know why this became needed?

Reply:
I don't know why it is necessary, but it doesn't work without it. Perhaps there is something different in the android fork of the kernel?